ValueError in _generate_device_id

Hi,

Sometimes, it seems that _generate_device_id raises a ValueError exception (see https://github.com/home-assistant/core/pull/40926/checks?check_run_id=1192119389)

```
>       mac_dec = int(random_bytes.replace(":", "").replace("-", ""), 16)
E       ValueError: invalid literal for int() with base 16: 'd4 d49085535'
```

This could be resolved by replacing spaces with "0"

--- a/surepy/__init__.py
+++ b/surepy/__init__.py
@@ -256,7 +256,7 @@ class SurePetcare:
             for i in range(0, 12, 2)
         )
 
-        mac_dec = int(random_bytes.replace(":", "").replace("-", ""), 16)
+        mac_dec = int(random_bytes.replace(":", "").replace("-", "").replace(" ", "0"), 16)
 
         # use low order bits because upper two octets are low entropy
         return str(mac_dec)[-10:]

